src/braket/experimental/autoqasm/api.py

```diff
@@ -177,7 +177,7 @@ def _add_qubit_declaration(program_conversion_context: aq_program.ProgramConvers
     Args:
         program_conversion_context (ProgramConversionContext): The program conversion context.
     """
-    root_oqpy_program = program_conversion_context.oqpy_program_stack[0]
```

Comment:
This correspond to the `aq.main`'s quantum program. Would be great to make naming consistent.